package.json

```diff
@@ -98,10 +98,13 @@
 	},
 	"dependencies": {
 		"@eventespresso/react-exit-modal-typeform": "^1.0.4",
-		"@wordpress/api-fetch": "^1.1.1",
-		"@wordpress/dom-ready": "^1.1.0",
-		"@wordpress/i18n": "^1.2.3",
-		"@wordpress/is-shallow-equal": "^1.0.2",
+		"@wordpress/api-fetch": "^2.0.0",
+		"@wordpress/dom-ready": "^2.0.0",
+		"@wordpress/hooks": "^2.0.0",
+		"@wordpress/i18n": "^2.0.0",
+		"@wordpress/is-shallow-equal": "^1.1.4",
+		"@wordpress/jest-preset-default": "^2.0.4",
+		"babel-runtime": "^6.26.0",
```

Comment:
was updating the versions in package.json necessary for this work?  This runs the risk of significant conflicts later on.  If you were just intending to do some housekeeping it would have been good to do it in the base branch and then merge in.

Reply:
I needed to upgrade "@wordpress/hooks" and thought I would do the rest as well.
Why would we *not* want to upgrade to the latest versions for everything when so many bug fixes and such are being added on a regular basis?

Reply:
I don't have a problem with you upgrading packages.  I have a problem that its done in this branch.  If you're updating packages it should be done in a separate pull and merged into the master branch (since that is something that can be done fairly quickly).  In this case it could have been upgraded directly in GB master then merged into your branch.  It matters because you increase the risk of tricky to fix merge conflicts.  

Now had you just installed a _new_  package that's okay, because the package is specific to what's being done in this branch.

The other issue here is that you've updated packages without also committing a `package-lock.json` file (which is yet another reason for not updating packages within the same branch as other work).  I really don't want to argue the point further though.  If you want to continue doing what you're doing question my suggestions we can just do that and then deal with the problems as they come.

Reply:
By the way, I'm not just pulling these suggestions out of my arse to make life difficult ;).  It seems to be a convention followed  in various projects I've contributed to so I figure there's good reason for it.